### PR TITLE
Cleaned Up Program.cs

### PR DESCRIPTION
**Pull request by openbots-ff**
Cleaned Up Program.cs

### CODE REVIEW

**Review by CameronHerwig — DISMISSED** [review]

File: taskt/Program.cs [hunk 1]
```diff
@@ -43,20 +44,21 @@ static void Main(string[] args)
             {
                 string filePath = args[0];
 
-
                 if (!System.IO.File.Exists(filePath))
```
Comment (ob-ab) [inline taskt/Program.cs]: Please cleanup `System.IO` namespace

File: taskt/Program.cs [hunk 2]
```diff
@@ -43,20 +44,21 @@ static void Main(string[] args)
...
                 {
                     using (System.Diagnostics.EventLog eventLog = new System.Diagnostics.EventLog("Application"))
```
Comment (ob-ab) [inline taskt/Program.cs]: Please cleanup `System.Diagnostics` namespace

**Review by ob-ab — COMMENTED** [review]
A couple of namespaces cleanup and everything else looks fine.

File: taskt/Program.cs (hunk 1 above)
Comment (openbots-ff) [reply taskt/Program.cs]: done

File: taskt/Program.cs (hunk 2 above)
Comment (openbots-ff) [reply taskt/Program.cs]: Done

**Review by ob-ab — APPROVED** [review]

**Review by CameronHerwig — APPROVED** [review]

**Pull request merged**